src/commands/retention-policies/create.js

```diff
@@ -46,6 +52,33 @@ RetentionPoliciesCreateCommand.flags = {
 		char: 'l',
 		description: 'The number of days to apply the retention policy. If not set, policy will be indefinite',
 	}),
+	'retention-type': flags.string({
+		description: 'The type of retention for the new policy',
+		exclusive: [
+			'modifiable',
+			'non-modifiable'
+		],
+		options: [
+			'modifiable',
+			'non_modifiable'
+		]
```

Comment:
Can you elaborate a bit on the reason for including the option to set modifiable as string? I thought this was done to allow new values without changing the code in the future (as it cannot be done with boolean flags), but I see that we are still limiting the possible options.

Reply:
I added this to be compatible with update method, when we can only update type to `non-modifiable`